app/controllers/assessments_controller.rb

```diff
@@ -131,7 +156,6 @@ def create
         # Create history if assessment was created by user
         History.report_created(patient: patient, created_by: current_user.email, comment: "User created a new report (ID: #{@assessment.id}).") if current_user
       end
-      redirect_to(patient_assessments_url)
```

Comment:
@holmesie I removed this because before it didn't seem to actually be doing anything (since `index` was empty) and now it is causing issues with it defined. Was there a reason for it before?

Reply:
The front end doesn't work this way anymore for assessments, so this really isn't needed.